src/components/playback/playbackmanager.js

```diff
@@ -4000,6 +4030,9 @@ define(['events', 'datetime', 'appSettings', 'itemHelper', 'pluginManager', 'pla
             case 'SetRepeatMode':
                 this.setRepeatMode(cmd.Arguments.RepeatMode, player);
                 break;
+            case 'SetQueueShuffleMode':
+                this.setQueueShuffleMode(cmd.Arguments.ShuffleMode, player);
+                break;
```

Comment:
Why not match `SetRepeatMode` and label this `SetShuffleMode` to keep it more generic for now? This is an excellent change though, I'm always a fan of better command support.

Reply:
@dkanada That's how it was called initially. However, didn't want to make a confusion between the "normal" shuffle (the one you run in the library browser, item details, context menu, etc) and the queue one. @jelyfin/web thoughts?

Reply:
I think all these should eventually be merged, so reusing the same name shouldn't be an issue.

Reply:
Were you going to add this in a future commit?

Reply:
@dkanada Is already done in b7714ee